### ISSUE TITLE
Auto-link email addresses do not render correctly.

### ISSUE THREAD

**Issue by alkalinethunder** [TobiasSN/vue-markdown-renderer#2]
![image](https://user-images.githubusercontent.com/31224503/83343354-35060500-a2c7-11ea-8d36-e501b744f3be.png)

While implementing @mentions support in my content management system, I have identified an issue where `mailto:` auto-links do not render correctly.  The above screenshot shows me entering "email@address.com" as a way to make sure the internal Marked tokenizer recognizes email addresses BEFORE user mentions, which thankfully it does.  However the link text renders with a bunch of HTML charactrer escape codes.

I've seen this issue before way back when `vue-markdown-renderer` was first created, and the bug was nipped in the butt almost right after I showed it.  Maybe it's a similar type of bug with email links?

Things I did to make sure it wasn't my code screwing up:

 - Disabled my @mention tokenizer completely
 - Tried writing a normal http:// link, works fine
 - Tried writing an https:// link, works fine

I also inspected the page HTML, and the generated HTML for that email link is as follows:

```
<a href="mailto:&amp;#101;&amp;#109;&amp;#x61;&amp;#105;&amp;#108;&amp;#64;&amp;#97;&amp;#100;&amp;#100;&amp;#114;&amp;#101;&amp;#115;&amp;#x73;&amp;#46;&amp;#x63;&amp;#111;&amp;#109;">&amp;#101;&amp;#109;&amp;#x61;&amp;#105;&amp;#108;&amp;#64;&amp;#97;&amp;#100;&amp;#100;&amp;#114;&amp;#101;&amp;#115;&amp;#x73;&amp;#46;&amp;#x63;&amp;#111;&amp;#109;</a>
```

So seems like it's messing up the `href` as well as the actual link text.

**Comment by TobiasSN** [comment]
I think this might be because Marked is automatically “mangling” emails, where it converts emails to that weird looking stuff. Normally, browsers would understand it and convert it back into the original text. However, the way Vue inserts text into the DOM (I assume by using `element.innerText`), the browser simply ignores the mangled text.

I will fix this when I get the chance, but until then, you can try setting `mangle` in the Marked config to `false`.

**Comment by TobiasSN** [comment]
Fixed in 0.2.6.
